src/popup/router/components/TransactionOverview.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <div class="transaction-overview">
+    <div class="title">
+      {{ transaction.title }}
+      <span class="description">({{ transaction.description }})</span>
```

Comment:
```suggestion
```
with @onvisions we [agreed](https://www.figma.com/file/ewTV7C01eFlKVODGHokUI0/Superhero-Wallet-UI?node-id=31%3A1533) to remove this